### ISSUE TITLE
Secured EJB lookup fails after AppClientContainer instantiation

### ISSUE THREAD

**Issue by glassfishrobot** [javaee/glassfish#21558]
We followed the documentation at [https://docs.oracle.com/cd/E19226-01/820-7695/gipkt/index.html](https://docs.oracle.com/cd/E19226-01/820-7695/gipkt/index.html) to set up a standalone ACC-Client application.

Although the callback-handler in the META-INF/application-client.xml is configured properly, the lookup for a secured SLSB fails and the handler is never called.

The lookup for unsecured SLSB works as expected.

```
Sep 07, 2016 12:28:26 PM com.sun.enterprise.iiop.security.SecurityMechanismSelector getUsernameAndPassword
SCHWERWIEGEND: IIOP1001: Exception getting username and password
java.lang.SecurityException: java.io.IOException: Konfigurationsfehler:
	Datei oder Verzeichnis nicht vorhanden
	at sun.security.provider.ConfigFile$Spi.<init>(ConfigFile.java:137)
	at sun.security.provider.ConfigFile.<init>(ConfigFile.java:102)
```
#### Affected Versions
[4.1.1]

**Comment by amykang2020** [comment]
This needs a reproducer application with more information for examination.

As this is for embedded AppClientContainer,  not  a must fix, if any issue, for GlassFish 5.0

**Comment by yaminikb** [comment]
Closing since we won't get response as filer does not exist on GitHub.
